config/set/contentrepository-90.php

```diff
@@ -379,9 +380,43 @@
     // Context::getNodesOnPath()
     // TODO: PHP
     // Context::adoptNode()
+    // TODO: PHP
     // Context::getFirstLevelNodeCache()
     $rectorConfig->rule(ContextGetFirstLevelNodeCacheRector::class);
+    // getCurrentDateTime(): DateTime|DateTimeInterface
+    // TODO: PHP
+    $fusionNodePropertyPathToWarningComments[] = new FusionNodePropertyPathToWarningComment('context.currentDateTime', 'Line %LINE: !! node.context.currentDateTime is removed in Neos 9.0.');
+    // getDimensions(): array
+    // TODO: PHP
+    $fusionNodePropertyPathToWarningComments[] = new FusionNodePropertyPathToWarningComment('context.dimensions', 'Line %LINE: !! node.context.dimensions is removed in Neos 9.0. You can get node DimensionSpacePoints via node.dimensionSpacePoints now.');
```

Comment:
also there is the `Neos.Dimension.*` helper now which does some things ... build for the seo module \Neos\Neos\Fusion\Helper\DimensionHelper
```suggestion
    $fusionNodePropertyPathToWarningComments[] = new FusionNodePropertyPathToWarningComment('context.dimensions', 'Line %LINE: !! node.context.dimensions is removed in Neos 9.0. You can get node DimensionSpacePoints via node.dimensionSpacePoints now or use the `Neos.Dimension.*` helper.');
```